source/model/Award.ts

```diff
@@ -0,0 +1,41 @@
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { IsInt, IsOptional, IsString } from 'class-validator';
+import { Base, Media } from './Base';
+import { Team } from './Team';
+import { User } from './User';
+import { HackathonBase } from './Hackathon';
+
+@Entity()
+export class Award extends HackathonBase {
+    @IsString()
+    @Column()
+    name: string;
+
+    @IsString()
+    @IsOptional()
+    @Column()
+    description: string;
+
+    @IsInt()
+    @IsOptional()
+    @Column()
+    quantity: number;
+
+    @IsString()
+    @IsOptional()
+    @Column()
+    target: 'team' | 'individual';
```

Comment:
声明 `enum AvwardTarget` 后：

```suggestion
    @IsEnum(AvwardTarget)
    @IsOptional()
    @Column({ type: 'simple-enum', enum: AwardTarget, nullable: true })
    target?: AwardTarget;
```